functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -45,9 +45,15 @@ var _ = BeforeSuite(func() {
 	// and should be greater than what is requested by the test cases in the suite so the test runs properly
 	var err error
 	profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
-	Expect(err).ToNot(HaveOccurred())
```

Comment:
Could you please explain why this change is needed?

Reply:
had this done in a separete commit to having the changes in this PR in groups. 
it was needed because it is not recommended to have Gomega assertions in the main ..Testing(t ..) function. it w